successstories/forms.py

```diff
@@ -24,6 +24,9 @@ class Meta:
         labels = {
             'name': 'Story name',
         }
+        help_texts = {
+            "content": "Note: Submissions in Markdown are strongly preferred and can be processed faster.",
```

Comment:
```suggestion
            "content": "Note: Submissions in <a href="https://www.markdownguide.org/basic-syntax/">Markdown</a> are strongly preferred and can be processed faster.",
```

Or some other introductory resource?

Reply:
I like that, considering target audience 

<img width="704" alt="image" src="https://github.com/user-attachments/assets/2f9b924b-3df1-4be1-b0b2-2965171182c9">

[cc85402](https://github.com/python/pythondotorg/pull/2594/commits/cc85402b3104dbc6892fb525c3d7ae9015d6600a)